website/content/api-docs/acl/roles.mdx

```diff
@@ -0,0 +1,340 @@
+---
+layout: api
+page_title: ACL Roles - HTTP API
+description: The /acl/roles endpoints are used to configure and manage ACL roles.
+---
+
+# ACL Roles HTTP API
+
+The `/acl/roles` and `/acl/role/` endpoints are used to manage ACL Roles.
+
+## Create Role
+
+This endpoint creates an ACL Role. The request is always forwarded to the
+authoritative region.
+
+| Method | Path        | Produces           |
+| ------ | ----------- | ------------------ |
+| `POST` | `/acl/role` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `management` |
+
+### Parameters
+
+- `Name` `(string: <required>)` - Specifies the human-readable name of the ACL
+  Role. The name can contain alphanumeric characters, dashes, and underscores.
+  This name must be unique and must not exceed 128 characters.
+
+- `Description` `(string: <optional>)` - A free form a human-readable, useful
+  description of the ACL Role. It must not exceed 256 characters.
+
+- `Policies` `(array<ACLRolePolicyLink>)` The list of policies that should be
+  applied to the role. An ACLRolePolicyLink is an object with a "Name" field to
+  specify a policy.
+
+### Sample Payload
+
+```json
+{
+  "Name": "example-acl-role",
+  "Description": "my example ACL Role",
+  "Policies": [
+    {
+      "Name": "example-acl-policy"
+    }
+  ]
```

Comment:
It would be possible, but if/when we add ID's to policies this would need to change and therefore it is done this way to accommodate future changes to the ACL system without breaking the API.

Reply:
Ahh OK. From an API perspective passing an object makes it seem like you can create the policy in the same request, but mixing IDs and names in a list doesn't sound great either 😅